specification/dimensions/commitmentid.md

```diff
@@ -0,0 +1,32 @@
+# Commitment ID
+
+A commitment-based discount is a contractual commitment to use a certain amount of a specific type of usage for a fixed term in exchange for a discounted unit price. Commitment-based discounts can cover either usage or cost of various resources such as virtual machines or databases. A provider can offer various pricing models, payment options, and term durations for commitment-based discounts.
+
+A Commitment ID is the identifier assigned to a commitment-based discount by the provider.
```

Comment:
Mention the scenarios that the column can be used to address, like commitment utilization and savings.

Reply:
Made an explicit suggestion to address this feedback in the latest iteration. Closing.